ccp/modules/id-management-compose.yml

```diff
@@ -57,7 +65,7 @@ services:
       - "/tmp/bridgehead/patientlist/:/docker-entrypoint-initdb.d/"
 
   traefik-forward-auth:
-    image: docker.verbis.dkfz.de/cache/oauth2-proxy/oauth2-proxy:v7.6.0
+    image: docker.verbis.dkfz.de/cache/oauth2-proxy/oauth2-proxy:latest
```

Comment:
Didn't Martin discuss explicit version tags at the department retreat? Is this applicable here?

Reply:
The change was actually introduced by Martin. Reasoning is, that we currently don't have the versions in one place and should introduce that with another PR.